v1.1/certificate-rotation.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Rotate Security Certificates
```

Comment:
We'll want a link from `Create Security Certificates` along with a quick mention of expiration and the need for rotation.
Possibly from other places as well, but definitely that one.